src/wordlist.h

```diff
@@ -12,6 +12,28 @@
 #include "globals.h"
 #include "sbuf/List.h"
 
+#include <iterator>
+
+class wordlist;
+
+class WordlistIterator
+{
+    const wordlist *w;
```

Comment:
Class layout style require this to be in an explicit `private:` section at the end of the class.

Reply:
added